configserver/src/main/java/com/yahoo/vespa/config/server/session/Session.java

```diff
@@ -27,13 +31,24 @@
     private final long sessionId;
     protected final TenantName tenant;
     protected final SessionZooKeeperClient sessionZooKeeperClient;
+    protected final Optional<ApplicationPackage> applicationPackage;
 
     protected Session(TenantName tenant, long sessionId, SessionZooKeeperClient sessionZooKeeperClient) {
         this.tenant = tenant;
         this.sessionId = sessionId;
         this.sessionZooKeeperClient = sessionZooKeeperClient;
+        this.applicationPackage = Optional.empty();
     }
 
+    protected Session(TenantName tenant, long sessionId, SessionZooKeeperClient sessionZooKeeperClient,
```

Comment:
This has duplicate code from the constructor above, but assuming it's temporary while refactoring the Session classes.

Reply:
Yes, will clean up